mail_editor/utils.py

```diff
@@ -4,8 +4,8 @@
 
 
 def variable_help_text(template_type):
-        subject_html = 'Subject: <br><ul>'
-        body_html = 'Body: <br><ul>'
+        subject_html = '<ul>'
```

Comment:
It is presented as a (single) field (so the label is added by the field template). But I guess we can use description again and revert this.